verticapy/sql/functions/analytic.py

```diff
@@ -240,6 +496,30 @@ def median(expr: SQLExpression) -> StringSQL:
     -------
     StringSQL
         SQL string.
+
+    Examples
+    --------
+    .. code-block:: python
+
+        from verticapy import TableSample
+        import verticapy.stats as st
+
+        df = tablesample({"x": [2, -11, 7, 12]}).to_vdf()
+        # apply the median function
+        df.select([st.median(df["x"])])
```

Comment:
this also seems to be recurring. you can put str() before such examples. 

df.select([str(st.median(df["x"]))])